pkg/disttask/importinto/subtask_executor.go

```diff
@@ -120,11 +123,27 @@ func postProcess(ctx context.Context, store kv.Storage, taskMeta *TaskMeta, subt
 	}
 
 	taskManager, err := storage.GetTaskManager()
+	if err != nil {
+		return err
+	}
+	if keyspace.IsRunningOnSystem() && config.GetGlobalKeyspaceName() != p.taskKeyspace {
+		var sp tidbutil.SessionPool
+		err = taskManager.WithNewSession(func(se sessionctx.Context) error {
+			svr := se.GetSQLServer()
+			sp, err = svr.GetKSSessPool(p.taskKeyspace)
+			return err
+		})
+		if err != nil {
+			return err
+		}
+		taskManager = storage.NewTaskManager(sp)
+	}
+
```

Comment:
i am working on the checksum part by using the checksum code of lightning

this change is invalid, we will forbid accessing user table in cross keyspace session later